hazelcast/src/main/java/com/hazelcast/config/WanPublisherState.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.config;
+
+/**
+ * Defines the state in which a WAN publisher can be in if it is not shutting
+ * down.
+ */
+public enum WanPublisherState {
+    /**
+     * State where both enqueuing new events is allowed, enqueued events are
+     * replicated to the target cluster and WAN sync is enabled.
+     */
+    ACTIVE((byte) 0, true, true),
```

Comment:
Is this the same `active` that we use when describing active-active/active-passive replication? Are they referring same things? Or for example, on a active-active replication mode, can we see a state like `WanPublisherState#PAUSED`?

Reply:
No, it's a different thing. We may call it "RUNNING" but we already have a `running` boolean in the publisher implementation.

Reply:
Maybe this can be an alternative naming:

```
public enum WanPublisherState {
    PUBLISHING((byte) 0, true, true),
    NOT_PUBLISHING((byte) 1, true, false),
    STOPPED((byte) 2, false, false);
```

Reply:
I'd rather not mix in the `publish` word. In the current code, we "publish" events (or enqueue them) both in the "paused" and in the "non-paused" state.
Actually, in the current code we very rarely use the ACTIVE-PASSIVE nomenclature. I've only seen it in tests and in the wiki/slack but not in the production code.

Reply:
Ideally, to implement synced and clear terminology would be helpful for docs, codes, zendesk communication, slack etc. 

I don't have another naming suggestion but i can only say, i found ACTIVE confusing and found PAUSED and STOPPED hard to differentiate.

Reply:
You're right. And we already had the first confusion - https://hazelcast.slack.com/archives/CB21BD4H0/p1528288312000215. I will rename `ACTIVE` to `REPLICATING` (or something like it) to eliminate any reference to ACTIVE-PASSIVE. I'm afraid one more reason why we can't rename `PAUSED` is that it is already used in the mancenter screens. Which leaves us with `STOPPED`. I don't have a good idea for that one. Maybe it's even good to keep it as `STOPPED` considering that we need to keep `PAUSED`. So you can think of it as some tape-recorder :)

Reply:
Renamed `ACTIVE` to `REPLICATING`. Now there isn't any connection to the ACTIVE-PASSIVE nomenclature. We can even rename it further in a separate PR if something better comes to mind.